apps/oc_chef_wm/priv/dispatch.conf

```diff
@@ -125,4 +125,11 @@
 {["system_recovery"], oc_chef_wm_system_recovery, []}.
 {["license"], oc_chef_wm_license, []}.
 
+%% keys
+%%
+{["organizations", organization_id, "clients", client_name, "keys" ], oc_chef_wm_keys, [{resource_args, client}]}.
+{["users", user_name, "keys" ], oc_chef_wm_keys, [{resource_args, user}]}.
```

Comment:
I was thinking about that. I'd like to just consistently apply whatever we do in the named user endpoint - most likely if the API version dictates it, return keys as a list of names and uris instead of the actual key value.  